recipes/unidiff/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "unidiff" %}
+{% set version = "0.5.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/matiasb/python-{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 5d351e402191b10e6b5164e6ce40b683f5f63eaedccc9a17d3ed47bbe3873634
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - unidiff
+  requires:
+    - python
```

Comment:
Not needed. Anything in the run requirements will get pulled in just fine.

Reply:
good. just a second.